include/votca/csg/beadlist.h

```diff
@@ -56,7 +57,7 @@ class BeadList {
   iterator begin() { return _beads.begin(); }
   iterator end() { return _beads.end(); }
 
-  Topology *getTopology() { return _topology; }
+  Topology *getTopology() const { return _topology; }
```

Comment:
want to return a pointer here or maybe a reference? And do we really want to return a "non const"?

Reply:
This was a loaded request... :)